lib/delayed_cron_job.rb

```diff
@@ -23,17 +23,12 @@ module DelayedCronJob
   Delayed::Backend::Mongoid::Job.send(:include, DelayedCronJob::Backend::UpdatableCron)
 end
 
-if defined?(Delayed::Backend::ActiveRecord)
-  if defined?(Rails::Railtie)
-    # Postpone initialization to railtie for correct order
-    require 'delayed_cron_job/backend/active_record/railtie'
-  else
-    # Do the same as in the railtie
-    Delayed::Backend::ActiveRecord::Job.send(:include, DelayedCronJob::Backend::UpdatableCron)
-    if Delayed::Backend::ActiveRecord::Job.respond_to?(:attr_accessible)
-      Delayed::Backend::ActiveRecord::Job.attr_accessible(:cron)
-    end
-  end
+if defined?(Rails::Railtie)
```

Comment:
To avoid this from being executed when mongoid is loaded, the following lines should probably go into an else block of the mongoid if.

Reply:
Moved that bit of code into the else block. Thanks :+1: